allennlp/modules/elmo.py

```diff
@@ -225,7 +228,7 @@ def from_params(cls, params: Params) -> 'Elmo':
                    scalar_mix_parameters=scalar_mix_parameters)
 
 
-def batch_to_ids(batch: List[List[str]]) -> torch.Tensor:
+def batch_to_ids(batch: List[List[str]]) -> Tuple[torch.Tensor, torch.Tensor]:
```

Comment:
After removing the vocab, all of the changes in this function should be reverted.